scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -317,6 +319,8 @@ function verify_subm_routeagent_pod() {
     kubectl get pod $subm_routeagent_pod_name --namespace=$subm_ns -o jsonpath='{.spec..volumes}' | grep "path:/"
     kubectl get pod $subm_routeagent_pod_name --namespace=$subm_ns -o jsonpath='{.status.phase}' | grep Running
     kubectl get pod $subm_routeagent_pod_name --namespace=$subm_ns -o jsonpath='{.metadata.namespace}' | grep $subm_ns
+    kubectl get pod $subm_routeagent_pod_name --namespace=$subm_ns -o jsonpath='{.spec.terminationGracePeriodSeconds}' | grep " 0"
```

Comment:
Wouldn't this technically also work for '30' since that also has 0?

Reply:
Right, I just changed it.

I thought the output was something like  "terminationGracePeriodSeconds: 0" and I was trying to rely on the front space, but it's not, so I sent another patch.